Makefile

```diff
@@ -105,6 +105,10 @@ build_docs_pdf:
 docs_serve: build_docs
 	podman run -it -v $(CURDIR)/docs:/docs:Z --rm -p 8088:8088/tcp $(DOCS_BUILD_CONTAINER) docs_serve
 
+.PHONY: check_links
+check_links:
+	podman run -it -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_BUILD_CONTAINER) check_links
```

Comment:
is the `-it` needed here?  Do the docs need to be built before we can check the links? or is it acting on the source asciidoc data?

Reply:
> is the -it needed here?

we get a nicer colored output and to stop the process inside the container by `^C`

>  Do the docs need to be built before we can check the links? or is it acting on the source asciidoc data?

No we don't need to build, its acting on The Source.